recipes-wpe/wpeframework/include/compositor.inc

```diff
@@ -12,7 +12,7 @@ WPE_COMPOSITOR_AUTOSTART 	?= "true"
 WPE_COMPOSITOR_OUTOFPROCESS ?= "true"
 WPE_COMPOSITOR_RESOLUTION	?= "720p"
 WPE_COMPOSITOR_MEMORY_GFX	?= ""
-WPE_COMPOSITOR_IRMODE		?= "16"
+WPE_COMPOSITOR_IRMODE		?= "NEXUS_IrInputMode_eCirGISat"
```

Comment:
this is BRCM specific. Why do we want it in generic variable ?

Reply:
Currently IRMODE is used with BRCM alone and its value is string. So using digit not configuring IRMODE properly

Reply:
I would think it would be better to have a "none" as default and then use WPE_COMPOSITOR_IRMODE_brcm = "NEXUS_IrInputMode_eCirGISat" will be more logical.

Reply:
Thanks @kraj, I will update accordingly